doc/README.cuda_support.md

```diff
@@ -16,6 +16,7 @@ is a reference table below.
 | Open-CE version | CUDA version |
 |-----------------|--------------|
 | 1.0 (marmotini) | 10.2         |
+| 1.1 (unnamed) | 10.2 \| 11.0  |
```

Comment:
Let's add a note below to explain a bit more on the dual support:
  - you can't do  `--cuda_versions 11.0,10.2`
  - for both, you'll have to run the build twice adjust CUDA_HOME
  - using the docker_build option will allow to build both on the same system, if the cuda11 driver is installed.

Reply:
If there's no way to build both at the same time, should we change that flag to be singular? I could see that leading to confusion.
@bnemanich

Reply:
I added more detail about using multiple versions here:
https://github.com/open-ce/open-ce/pull/243/files#diff-cb96e2b228ce0a549bef55d3963cc18ee6540ecef9c6b85bc4cd6312c7acc108R96-R102